packages/insured-bridge-relayer/src/Relayer.ts

```diff
@@ -33,6 +31,8 @@ export enum RelaySubmitType {
 }
 
 export class Relayer {
+  executedTransactions: Array<executedTransaction> = []; // store all submitted transactions during execution lifecycle.
```

Comment:
this is part of the main change in this PR: store all `executedTransaction`s in an array that we can then `await` at the end of the execution loop.